### PR TITLE
LIB-718: add structure and most stylings for multi step code snippet

### PR DESCRIPTION
**Pull request by oriooctopus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "@docusaurus/preset-classic": "^2.0.0-alpha.53",
     "@svgr/webpack": "^5.4.0",
     "classnames": "^2.2.6",
-    "libra-docusaurus": "^0.4.7",
+    "libra-docusaurus": "^0.4.10",
```
Comment (jjwill) [inline package.json:17]: Let's rename this to libra-components. The name could be confused with a custom implementation of docusaurus.

**Review by jjwill — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (oriooctopus) [reply package.json:17]: Adding this as a todo, since I feel like it should be it's own commit

Comment (oriooctopus) [reply package.json:17]: https://dfuzrdev.atlassian.net/browse/LIB-721

**Pull request closed without merging**